rpcs3/Emu/Cell/Modules/sceNpClans.h

```diff
@@ -225,8 +225,8 @@ struct SceNpClansMemberEntry
 // Clan message structure
 struct SceNpClansMessage
 {
-	s8 subject[SCE_NP_CLANS_MESSAGE_SUBJECT_MAX_LENGTH + 1];
-	s8 body[SCE_NP_CLANS_MESSAGE_BODY_MAX_LENGTH + 1];
+	char subject[SCE_NP_CLANS_MESSAGE_SUBJECT_MAX_LENGTH + 1];
+	char body[SCE_NP_CLANS_MESSAGE_BODY_MAX_LENGTH + 1];
```

Comment:
Why the change here?

Reply:
Can't use strlen on s8.
And it's clear that these are supposed to be chars and this was probably wrong in the first place.